usgs-details.html

```diff
@@ -182,35 +180,55 @@
         }
 
         // Create header text for live and fallback values
-        function renderHeader(metric, isFallback) {
-            var firstWord = isFallback ? 'Typical ' : 'Current ';
-            return firstWord + metric.display + ' in the Schuylkill River:';
+        function renderHeader(metric, isFallback, isBackupSensor) {
```

Comment:
It's starting to feel like we're passing too many parameters too often. Since `isFallback` and `isBackupSensor` are essentially global state values, consider making them so, setting them once, and using them directly, instead of passing in each call.